docs/examples/table/dontAddSoMuchContent.js

```diff
@@ -0,0 +1,66 @@
+// @flow strict
+import { type Node } from 'react';
+import { Table, Text } from 'gestalt';
+
+function HeaderRow() {
+  return (
+    <Table.Header>
+      <Table.Row>
+        {['Name', 'Total amounts for 2021 through 2022'].map((title) => (
+          <Table.HeaderCell key={title}>
+            <Text align={title.startsWith('Total') ? 'end' : 'start'} weight="bold">
+              {title}
+            </Text>
+          </Table.HeaderCell>
+        ))}
+      </Table.Row>
+    </Table.Header>
+  );
+}
+
+// $FlowIgnore
```

Comment:
@AlbertCarreras @dangerismycat Ignored flow type errors in "Don't" examples as the source code is not presented.